React-Core.podspec

```diff
@@ -70,14 +70,14 @@ Pod::Spec.new do |s|
   end
 
   s.subspec "Hermes" do |ss|
-    ss.platforms = { :osx => "10.14" }
+    ss.platforms = { :osx => "10.14", :ios => "10.0" }
     ss.source_files = "ReactCommon/hermes/executor/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/chrome/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/detail/*.{cpp,h}"
     ss.pod_target_xcconfig = { "GCC_PREPROCESSOR_DEFINITIONS" => "HERMES_ENABLE_DEBUGGER=1" }
     ss.dependency "RCT-Folly/Futures"
-    ss.dependency "hermes", "~> 0.6.0"
+    ss.dependency "hermes"
```

Comment:
I think since `hermes` dependency is always pulled from the local `npm` package, I don't think it makes sense to a) keep version requirement in two places and b) read a version from `package.json` here. The main reason against doing the latter is that CocoaPods has slightly different (and incompatible with npm) syntax for specifying the minimum supported version.

Since this package is part of React Native where we do have control over the Hermes version (and set a requirement on it inside `package.json`), I decided to strip it entirely.

Reply:
CC: @alloy

Reply:
Yep, makes sense 👍